lib/TickLib.sol

```diff
@@ -209,7 +209,9 @@ library TickLib {
   // see note posIn*
   // note with int24 tick we only use 64 bits of level2 (64*256*256*4 is 2**24)
   // the goal is that have the bit positions in {} used:
-
+// FIXME: This is a non-standard way of writing bit positions, normally they're numbered right-to-left
+// FIXME: Also, the highest position in a field is 63 and is used for the highest tick in the level.
+//        This feels like it's conflicting with the stmt in the Excalidraw that says "ticks go right (cheap) to left (expensive)"
```

Comment:
Maybe we can leave this comment here for when we update all comments?
@adhusson wdyt?

Reply:
Yes the diagram is in reverse, it should go the other way, so we can leave the comment there and reverse it later!